Builds/VisualStudio/stellar-core.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28803.202
+# Visual Studio Version 17
+VisualStudioVersion = 17.1.32421.90
```

Comment:
why did this change? (so far I don't think changes caused this bump)

Reply:
Must be some automatic VS stamp, reverted.